src/views/Claims.vue

```diff
@@ -98,10 +126,21 @@ export default class Claims extends Vue {
             from: this.account,
         });
     }
+    async insetWallet() {
+        const postURl = 'http://localhost:3001/v1/claims/wallet';
```

Comment:
Move base URL to ENV, it is easier to change it in 1 location instead of having to search for multiple mentions.
The endpoint (/claims/wallet) will most likely remain the same.

Look at the for example the .env.local where VUE_APP_API_URL is present.